MultipleFileUpload/Model/MySQL/Queue.php

```diff
@@ -0,0 +1,162 @@
+<?php
```

Comment:
I check just right now and for sure it is not compatible with PHP 7 so I will continue in update.